.atomist/handlers/command/issue/ListIssues.ts

```diff
@@ -106,7 +162,7 @@ class ListIssuesRender implements HandleResponse<Issue[]> {
 }
 
 const command = new ListIssuesCommand();
-const render = new ListIssuesRender();
+const issueList = new ListIssuesRender();
 const repo = new ListRepositoryIssuesCommand();
 
-export { command, render, repo };
+export { command, issueList, repo };
```

Comment:
We've been moving to exporting when defining, which appears to be the standard in TypeScript.